pkg/flashdb/include/fal_cfg.h

```diff
@@ -26,11 +26,34 @@
 #define FAL_CFG_H
 
 #include "board.h"
+#include "mtd_default.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#if !defined(CONFIG_FLASHDB_MIN_SECTOR_SIZE_EXP) || defined(DOXYGEN)
+/**
+ * @brief   Virtual sector size exponent for FlashDB
+ *
+ * The default is 2¹² (4096).
+ * FlashDB sector size is different from MTD sector size as it is a
+ * virtual measure of granularity and not a device property.
+ * The virtual sector size must be a multiple of the physical sector size.
+ *
+ * From the documentation of FLashDB:
+ * The internal storage structure of FlashDB is composed of N sectors, and each formatting takes
+ * sector as the smallest unit. A sector is usually N times the size of the Flash block.
+ * For example, the block size of Nor Flash is generally 4096.
+ *
+ * By default, KVDB will use 1 times the block size as the sector size, that is, 4096.
+ * At this time, the KVDB cannot store a KV longer than 4096. If you want to save, for example,
+ * a KV with a length of 10K, you can use the control function to set the sector size to 12K or larger.
+ *
+ */
+#define CONFIG_FLASHDB_MIN_SECTOR_SIZE_EXP      12
```

Comment:
I should say that this is just the default but it can be different for 2 databases.
For that I would rename this to `CONFIG_FLASHDB_DEFAULT_MIN_SECTOR_SIZE_EXP`.

A module `M` that initializes a database should have something like:
```C
#ifndef M_FLASHDB_MIN_SECTOR_SIZE_EXP
#define M_FLASHDB_MIN_SECTOR_SIZE_EXP CONFIG_FLASHDB_MIN_SECTOR_SIZE_EXP
#endif
```

Do you agree?

Hmm and the fact that it can be 10K or 12 actually means that it is not necessarily a power of 2.
I would change it to a factor `x KiB` Default would be 4 for 4K

Reply:
> the fact that it can be 10K or 12 

Ah ok so there are no assumptions about this being a power of two in the code then - where did you find the 10k sector?

Reply:
In the documentation of now ` CONFIG_FLASHDB_MIN_SECTOR_SIZE_DEFAULT_FACTOR` I copied something from the [documentation of flashDB [Sector size and block size](https://armink.github.io/FlashDB/#/api?id=sector-size-and-block-size).